aiven/client/cli.py

```diff
@@ -1197,25 +1197,30 @@ def service__migration_status(self):
             table_layout=layout,
         )
 
-    def _parse_redis_access_control(self):
+    def _parse_access_control(self):
         arg_vars = vars(self.args)
-        return {
+        result = {
             key: arg_vars[key].split()
             for key in {"redis_acl_keys", "redis_acl_commands", "redis_acl_categories"}
             if arg_vars[key] is not None
         }
+        for key in ["m3_group"]:
+            if key in arg_vars:
```

Comment:
Isn't `m3_group` always in the args (with value `None` if `--m3-group` not given)? Is the idea to always include it in the result (in which case this `if` is redundant), or only when the flag is given (in which case it could be handled in the dict comprehension above)?